src/main.rs

```diff
@@ -70,6 +90,14 @@ fn main() -> Result<()> {
         scope.raw("#![allow(clippy::wrong_self_convention)]");
         scope.raw("#![allow(clippy::large_enum_variant)]");
         scope.raw("#![allow(clippy::unreadable_literal)]");
+
+        // Adding getter for descriptor pool
+        let line = format!("static DESCRIPTOR_POOL: Lazy<DescriptorPool>
+        = Lazy::new(|| DescriptorPool::decode(include_bytes!(\"{}/descriptor.bin\").as_ref()).unwrap());", DESCRIPTOR_PATH);
```

Comment:
Could we move this after the imports in this block? Would make it more similar to the source code order.